CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ For details about compatibility between different releases, see the **Commitment
 ### Fixed
 - CLI command `end-device template create` no longer breaks when providing field mask values.
 - Device repository services no longer require ApplicationID in its request URL.
+- TTSCSV import no longer break when providing Session key values.
```

Comment:
`Importing ABP devices via the CSV format now correctly handles the missing session key ID`.